tests/cli/main.cpp

```diff
@@ -505,65 +474,25 @@ BOOST_AUTO_TEST_CASE( cli_confidential_tx_test )
       BOOST_CHECK(dgp.head_block_number == head_block);
-
-      // wait for everything to finish up
-      fc::usleep(fc::seconds(1));
    } catch( fc::exception& e ) {
       edump((e.to_detail_string()));
       throw;
    }
-   app1->shutdown();
 }
 
 /******
  * Check account history pagination (see bitshares-core/issue/1176)
  */
-BOOST_AUTO_TEST_CASE( account_history_pagination )
+BOOST_FIXTURE_TEST_CASE( account_history_pagination, cli_fixture )
 {
-   using namespace graphene::chain;
-   using namespace graphene::app;
-   std::shared_ptr<graphene::app::application> app1;
    try
    {
-      fc::temp_directory app_dir ( graphene::utilities::temp_directory_path() );
-
-      int server_port_number = 0;
-      app1 = start_application(app_dir, server_port_number);
-
-      // connect to the server
-      client_connection con(app1, app_dir, server_port_number);
-
-      // set wallet password
-      BOOST_TEST_MESSAGE("Setting wallet password");
-      con.wallet_api_ptr->set_password("supersecret");
-      con.wallet_api_ptr->unlock("supersecret");
-
-      // import Nathan account
-      BOOST_TEST_MESSAGE("Importing nathan key");
-      std::vector<std::string> nathan_keys{"5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3"};
-      BOOST_CHECK_EQUAL(nathan_keys[0], "5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3");
-      BOOST_CHECK(con.wallet_api_ptr->import_key("nathan", nathan_keys[0]));
-
-      BOOST_TEST_MESSAGE("Importing nathan's balance");
-      std::vector<signed_transaction> import_txs = con.wallet_api_ptr->import_balance("nathan", nathan_keys, true);
-      account_object nathan_acct_before_upgrade = con.wallet_api_ptr->get_account("nathan");
-
-      // upgrade nathan
-      BOOST_TEST_MESSAGE("Upgrading Nathan to LTM");
-      signed_transaction upgrade_tx = con.wallet_api_ptr->upgrade_account("nathan", true);
-      account_object nathan_acct_after_upgrade = con.wallet_api_ptr->get_account("nathan");
-
-      // verify that the upgrade was successful
-      BOOST_CHECK_PREDICATE( std::not_equal_to<uint32_t>(),
-                             (nathan_acct_before_upgrade.membership_expiration_date.sec_since_epoch())
-                             (nathan_acct_after_upgrade.membership_expiration_date.sec_since_epoch()) );
-      BOOST_CHECK(nathan_acct_after_upgrade.is_lifetime_member());
+      INVOKE(upgrade_nathan_account);
 
       // create a new account
       graphene::wallet::brain_key_info bki = con.wallet_api_ptr->suggest_brain_key();
       BOOST_CHECK(!bki.brain_priv_key.empty());
-      signed_transaction create_acct_tx = con.wallet_api_ptr->create_account_with_brain_key(
-                                                bki.brain_priv_key, "jmjatlanta", "nathan", "nathan", true);
+      signed_transaction create_acct_tx = con.wallet_api_ptr->create_account_with_brain_key(bki.brain_priv_key, "jmjatlanta", "nathan", "nathan", true);
```

Comment:
Apparently your screen is wider than mine. I like the wrapped format, at max around 130 characters a line.

Reply:
ok sure ! I will format the whole file right now as required !